README.md

```diff
@@ -8,15 +8,15 @@ Tokens will be validated for:
 * Expiration
 * Signature
 * Valid issuer
-* Valid Client/Audience
+* Valid Audience
 
 Basic usage:
 
 ``` java
 // 1. build the parser
 JwtVerifier jwtVerifier = new JwtHelper()
                             .setIssuerUrl("https://dev-123456.oktapreview.com/oauth2/ausar5cbq5TRooicu812")
-                            .setClientOrAudience("my-audience")
+                            .setAudience("my-audience")
```

Comment:
The plan is to just talk about the access token validation for now, and possibly a footnote about the id token validation, it would be a pretty specific use case to need to do id token validation outside of an existing framework